ui/src/templates/components/CommunityTemplateContents.tsx

```diff
@@ -196,10 +192,10 @@ class CommunityTemplateContentsUnconnected extends PureComponent<Props> {
         </CommunityTemplateListGroup>
         <CommunityTemplateListGroup
           title="Labels"
-          count={getResourceInstallCount(activeCommunityTemplate.labels)}
+          count={getResourceInstallCount(summary.labels)}
         >
-          {Array.isArray(activeCommunityTemplate.labels) &&
-            activeCommunityTemplate.labels.map(label => {
+          {Array.isArray(summary.labels) &&
+            summary.labels.map(label => {
```

Comment:
I wonder if you can apply defaults before `summary` enters the redux store so you don't have to check `Array.isArray` every time you access these parameters?

Reply:
good callout, i think at this point, these checks aren't necessary. i'll look into ditching them